.github/workflows/docker-stack-test.yml

```diff
@@ -0,0 +1,150 @@
+name: Docker-Stack-Test
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - Docker-Stack-Testing
+
+
+    tags:
+      - '*'
+
+  pull_request:
+
+  merge_group:
+
+  issue_comment:
+    types:
+      - created
+
+env:
+  R_LIBS_USER: /usr/local/lib/R/site-library
+  LC_ALL: en_US.UTF-8
+  NCPUS: 2
+  PGHOST: postgres
+  CI: true
+
+jobs:
+ 
+
+  # ----------------------------------------------------------------------
+  # DOCKER STACK TESTS
+  # ----------------------------------------------------------------------
+  Stack-Test:
+    if: github.event_name != 'issue_comment' || startsWith(github.event.comment.body, '/build')
```

Comment:
Does the `/build` comment command need to run these tests? If not, these `if`s could likely be dropped.

Reply:
Updated the file accordingly :)